### ISSUE TITLE
(Probably bug) Can't disable file shuttle on Android 10

### ISSUE THREAD

**Issue by CryptKid** [PeterCxy/Shelter#147]
I have managed to get shelter into a state i can neither disable nor use file shuttle on my Phone.
I know File Shuttle does not work on Android 10, but that is not my issue.
Steps to reproduce:
Use a Smartphone with Android 9 (lineageOS in my case), install shelter, enable file shuttle, upgrade to Android 10
The slider is still in the "on" possion but greyed out, so i can't disable file shuttle to get rid of the not-working option in files explorer

This is probably a bug as greying out is supposed to prevent people from activating file shuttle not from deactivating

**Comment by Lvl4Sword** [comment]
I'm not sure what "File Shuttle" is, but if it doesn't work on 10, that sounds more like an app issue as opposed to a Shelter issue.

**Comment by PeterCxy** [comment]
Indeed, I haven't really considered the upgrade scenario very much since I myself almost never upgrade Android versions without formatting the entire data partition. I will look into this later but I think I should probably try to implement File Shuttle again first.

By the way, the GitHub issue list is now deprecated, and all new issues should go to the new mailing list. I am going to close the issues entirely once most outstanding bugs have been fixed.
